openedx/core/djangoapps/safe_sessions/middleware.py

```diff
@@ -284,7 +286,18 @@ def verify(self, user_id):
             ).unsign_object(self.signature, serializer=signing.JSONSerializer, max_age=settings.SESSION_COOKIE_AGE)
 
             if unsigned_data == self._compute_digest(user_id):
+                set_custom_attribute('SafeCookieData.verify.alg', settings.DEFAULT_HASHING_ALGORITHM)
```

Comment:
Nit: But this will make it more clear what line of code is running. You should have `settings.DEFAULT_HASHING_ALGORITHM` in another attribute, so you won't be losing data.
Also, I changed `verify` to `verified`, because this was actually verified, not just used to verify.
```suggestion
                set_custom_attribute('SafeCookieData.verified.alg', 'default')
```